Comanche055/REENTRY_CONTROL.agc

```diff
@@ -1463,7 +1463,7 @@ VREFER		DEC	.019288		# REFERENCE VELOCITY SCALED V/51532.3946
 # DEFINED BY EQUALS
 
 DEC15		=	LOW4
-#GAMMAL1	=	22D
+GAMMAL1		=	22D
```

Comment:
In split diff view, the spacing looks wrong, but in [full file](https://github.com/abrogley/Apollo-11/blob/84cb4f66477952e360974d800decc6b0841fcacf/Comanche055/REENTRY_CONTROL.agc#L1466) view it looks okay

Reply:
Must be the leading `+` that is messing it up.